### ISSUE TITLE
Feature Request to show in right side bar

### ISSUE THREAD

**Issue by sethfair** [gidongkwon/logseq-plugin-tags#2]
Hi, this is such an excellent plugin! Thank you for making it.  

One issue I came across is when the popup window is visible. This feature is great for finding things on the fly, but when using it as a research tool and switching between documents, the popup usually gets in the way of the content. It then needs to be closed to be able to read the page. 

It would be wonderful if the plugin were visible in the right sidebar. It would be out of the way of the content pane and could also remain visible.

**Comment by gidongkwon** [comment]
Yes, I also think it'll be great if we can use right sidebar area. But, I think it's currently impossible by plugin api design. Do you know any other plugin that implements features like that?
